releases/release-5.3.0.md

```diff
@@ -0,0 +1,395 @@
+
+    [User Document](/tidb-lightning/tidb-lightning-prechecks.md)
+
+- **TiDB Lightning supports importing files of GBK character set**
+
+    You can specify the character set of the source data file. TiDB Lightning will convert the source file from the specified character set to UTF-8 encoding during the import process.
+
+    [User Document](/tidb-lightning/tidb-lightning-configuration.md)
+
+- **Sync-diff-inspector improvement**
+
+    - Improve the comparison speed from 375 MB/s to 700 MB/s
+    - Reduce the memory consumption of TiDB nodes by nearly half during comparison
+    - Optimize the user interface and display the progress bar during comparison
+
+    [User Document](/sync-diff-inspector/sync-diff-inspector-overview.md)
+
+### Diagnostic efficiency
+
+- **Save and restore the on-site information of a cluster**
+
+    When you locate and troubleshoot the issues of a TiDB cluster, you often need to provide information on the system and the query plan. To help you get the information and troubleshoot cluster issues in a more convenient and efficient way, the `PLAN REPLAY` command is introduced in TiDB v5.3.0. This command enables you to easily save and restore the on-site information of a cluster, improves the efficiency of troubleshooting, and helps you more easily archive the issues for management.
+
+    The features of `PLAN REPLAYER` are as follows:
+
+    - Exports the information of a TiDB cluster at an on-site troubleshooting to a ZIP-formatted file for storage.
+    - Imports into a cluster the ZIP-formatted file exported from another TiDB cluster. This file contains the information of the latter TiDB cluster at an on-site troubleshooting.
+
+    [User document](/sql-plan-replayer.md), [#26325](https://github.com/pingcap/tidb/issues/26325)
+
+### TiDB data share subscription
+
+- **TiCDC Eventually Consistent Replication**
+
+    TiCDC provides the eventually consistent replication capability in disaster scenarios. When a disaster occurs in the primary TiDB cluster and the service cannot be resumed in a short period of time, TiCDC needs to provide the ability to ensure the consistency of data in the secondary cluster. Meanwhile, TiCDC needs to allow the business to quickly switch the traffic to the secondary cluster to avoid the database being unavailable for a long time and affecting the business.
+
+    This feature supports TiCDC to replicate incremental data from a TiDB cluster to the secondary relational database TiDB/Aurora/MySQL/MariaDB. In case the primary cluster crashes, TiCDC can recover the secondary cluster to a certain snapshot in the primary cluster within 30 minutes, given the condition that before disaster the replication status of TiCDC is normal and replication lag is small. It allows data loss of less than 5 minutes, that is, RPO <= 30min, and RTO <= 5min.
+
+    [User Document](/ticdc/manage-ticdc.md)
+
```

Comment:
```suggestion

- **TiCDC supports the HTTP protocol OpenAPI for managing TiCDC tasks**
    Since TiDB v5.3.0, OpenAPI provided by TiCDC becomes an official feature. You can query and operate TiCDC clusters using OpenAPI.
    
```